pennylane/tape/qscript.py

```diff
@@ -280,7 +288,7 @@ def operations(self) -> List[Operator]:
         >>> qscript.operations
         [StatePrep([0, 1], wires=[0]), RX(0.432, wires=[0])]
         """
-        return self._prep + self._ops
+        return self._ops
```

Comment:
We may need to return a shallow copy to `_ops` instead of `_ops` itself. I'm seeing problems in lightning with `dev.apply` modifying the tape in place.

Reply:
chatted offline, probably leaving this as-is and fixing it in lightning.